Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h

```diff
@@ -257,9 +255,15 @@ compute_vertex_normal(typename boost::graph_traits<PolygonMesh>::vertex_descript
   {
     if (!is_border(he, pmesh))
     {
-      Vector n = fnmap_valid ? get(fnmap, face(he, pmesh))
-                             : compute_face_normal(face(he, pmesh), pmesh, np);
-      normal = traits.construct_sum_of_vectors_3_object()(normal, n);
+      Vector v1 = vector_3(get(vpmap, v /*target(he, pmesh)*/),
+                           get(vpmap, source(he, pmesh)));
+      Vector v2 = vector_3(get(vpmap, v /*target(he, pmesh)*/),
+                           get(vpmap, target(next(he, pmesh), pmesh)));
+      //v(i) and v(i+1) must me seen in ccw order, from v, so we reverse v1 and v2
+      Vector ni = traits.construct_cross_product_vector_3_object()(v2, v1);
+      ni = traits.construct_scaled_vector_3_object()(ni, CGAL::sqrt(1./(sqlen(v1)*sqlen(v2))));
```

Comment:
right. I should use `CGAL;;approximate_sqrt`. 
Just a remark : the documentation says that, even with an exact kernel, the sqrt inside will involve non exact computations.

And No, I did not get an email from Travis.